skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,349 @@
+            using ** for exponents instead of ^
+
+        Arithmetic operators:
+            + : addition (adds values on either side of the operator
+            - : subtraction (subtracts values on either side of the operator
+            * : multiplication (multiplies values on either side of the
+                operator
+            / : division (divides the left operand (numerator) by the right
+                hand operand (denominator))
+            % : modulus (divides the left operand (numerator) by the right
+                hand operand (denominator) and returns the remainder)
+            ** : exponent (left operand (base) is raised to the power of the
+                 right operand (exponent))
+            // : floor division (divides the left operand (numerator) by the
+                 right hand operand (denominator), but returns the quotient
+                 with any digits after the decimal point removed,
+                 e.g. 9.0/2.0 = 4.0)
+
+        Logical operations are also included and available so long as the:
+            > : greater than
+            < : less than
+            == : exactly equals
+            != : not equal
+            >= : greater than or equal
+            <= : less than or equal
+
+        Additional operators:
+            = : assignment operator (assigns values from right side to those
+                on the left side)
+            += : Adds the right operand to the left operand and sets the
+                 total equal to the left operand,
+                 e.g.:
+                    b+=a is equivalent to b=a+b
+            -= : Subtracts the right operand from the left operand and sets
+                 the total equal to the left operand,
+                 e.g.:
+                    b -= a is equivalent to b = b - a
+            *= : multiplies the right operand to the left operand and sets the
+                 total equal to the left operand,
+                 e.g.:
+                    b *= a is equivalent to b = b * a
+            /= : divides the right operand into the left operand and sets the
+                 total equal to the left operand,
+                 e.g.:
+                    b /= a is equivalent to b = b / a
+            %= : divides the right operand into the left operand and sets the
+                 remainder equal to the left operand,
+                 e.g.:
+                    b %= a is equivalent to b =b % a
+            **= : raises the left operand to the power of the right operand
+                  and sets the total equal to the left operand,
+                  e.g.:
+                    b **= a is equivalent to b = b ** a
+            //= : divides the right operand into the left operand and
+                  then removes any values after the decimal point. The total
+                  is then set equal to the left operand,
+                 e.g.:
+                    b //= a is equivalent to b = b // a

```

Comment:
The equations we attempted to use to test these operators were similar to 3 += 4. Since the additional operators (+=, -=, %=, etc) inherently change the value of the left hand parameter, a constant will fail. The left hand parameter must be a variable. Thus all of these operations are expected to work in this function. Tests will be added to the test function in order to confirm, verify and validate functionality.